gtfs_realtime_translators/bindings/intersection.proto

```diff
@@ -29,3 +29,12 @@ extend transit_realtime.TripUpdate {
 extend transit_realtime.TripUpdate.StopTimeUpdate {
   optional IntersectionStopTimeUpdate intersection_stop_time_update = 1987;
 }
+
+
+message IntersectionVehicleDescriptor {
+  optional uint32 run_number = 1;
+}
+
+extend transit_realtime.VehicleDescriptor {
```

Comment:
Is there any reason you chose to extend `transit_realtime.VehicleDescriptor` instead of `transit_realtime.TripUpdate.VehicleDescriptor`?

Reply:
In `transit_realtime` the `VehicleDescriptor` is a separate [message](https://github.com/google/transit/blob/eb71ab1ce08857ed9b8b9e082a1f4c794c7c2d1c/gtfs-realtime/proto/gtfs-realtime.proto#L881)

Reply:
Got it, thanks for clarifying.